Presentation/Borrow/Render/BorrowConsoleRenderer .cs

```diff
@@ -5,21 +5,21 @@ namespace Opcion1LosBorbotones.Presentation;
 
 public class BorrowConsoleRenderer : IBorrowConsoleRenderer
 {
-    public Guid GetPatronId()
+    public long GetMembershipNumber()
     {
-        string patronId = AnsiConsole.Ask<string>("What patron id do you want to register?: ");
-        return new Guid(patronId);
+        long patronId = AnsiConsole.Ask<long>("What patron Membership Number do you want to register?: ");
```

Comment:
You can do the same with the loan selections.

Reply:
done